test/service/test_request_validation.py

```diff
@@ -29,6 +30,15 @@ def lambda_name(cls) -> str:
                           "Message": "BadRequestError: Unable to sort by undefined facet bad-facet."}
     service_config_dir = os.path.dirname(service_config.__file__)
 
+    @classmethod
```

Comment:
Not a good idea to duplicate production code this way. Try to make the assertions that use this method less strict by asserting a fragment of the response. That will reduce the brittleness of the tests that make those assertions.